NOpenCL/CommandQueue.cs

```diff
@@ -8,7 +8,10 @@ namespace NOpenCL
     using System;
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
+    using System.Diagnostics;
+    using System.Runtime.InteropServices;
 
+    [DebuggerDisplay("Device = {Device.Name}")]
```

Comment:
I have tested it and it worked great for me. I did not see any side-effects when I using it.  It made it much easier to identify what the device is.  

I'll try and send this in a separate pull request.  I'm still learning Git.  Since this is its own item should it normally be in its own pull request? (vs sending one pull request with a boat load of items like I did)